pkg/proxy/backup.go

```diff
@@ -357,7 +358,20 @@ func (p *Proxy) BackupConfigMetaGet(ctx context.Context, req *rpc.EngineBackupCo
 
 	resp = &rpc.EngineBackupConfigMetaGetProxyResponse{}
```

Comment:
NIT: same here. don't need to init the response at the beginning.

Reply:
Updated.